core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -2466,6 +2467,34 @@ class GroupMetadataManagerTest {
       verifySerde(version)
   }
 
+  @Test
```

Comment:
Should we add tests which verify that the coordinators can deserialize the newest versions? I also wonder if we could craft a newer version with tagged fields somehow. I am not sure if this is even possible.